script.js

```diff
@@ -1,4 +1,146 @@
+      input == "stone" ||
+      input == "reverse scissors" ||
+      input == "reverse paper" ||
+      input == "reverse stone"
+    )
+  ) {
+    console.log("enter again");
+    return `Pls choose again! <br><br> You can only choose 'scissors✌', 'paper🖐' or 'stone👊'!`;
+  }
+
+  // User input win conditions
+  if (
+    (input == "scissors" && computerChoice == 2) ||
+    (input == "paper" && computerChoice == 3) ||
+    (input == "stone" && computerChoice == 1) ||
+    (input == "reverse scissors" && computerChoice == 3) ||
+    (input == "reverse paper" && computerChoice == 1) ||
+    (input == "reverse stone" && computerChoice == 2)
+  ) {
+    console.log(input);
+    console.log(computerChoice);
+    console.log("win!");
+    numOfUserWins += 1;
+    numOfComLosses += 1;
+    totalNumOfRounds += 1;
+    message += `<br> ${userName}, You chose ${input}. <br><br> You win! <br><br> (Number of User wins : Number of Computer losses) = ${numOfUserWins}: ${numOfComLosses}. <br><br> so far ${userName}, you've been winning ${numOfUserWins}/${totalNumOfRounds} turns. Pretty good!`;
+    return message;
+  }
+
+  // Draw conditions
+  if (
+    (input == "scissors" && computerChoice == 1) ||
+    (input == "paper" && computerChoice == 2) ||
+    (input == "stone" && computerChoice == 3) ||
+    (input == "reverse scissors" && computerChoice == 1) ||
+    (input == "reverse paper" && computerChoice == 2) ||
+    (input == "reverse stone" && computerChoice == 3)
+  ) {
+    console.log(input);
+    console.log(computerChoice);
+    console.log("draw!");
+    numOfDraws += 1;
+    totalNumOfRounds += 1;
+    message += `<br>${userName}, You chose ${input}. <br><br> It's a draw! <br><br> Try again!<br><br> Number of Draws is ${numOfDraws}.`;
+    return message;
+  }
+
+  // User input lose conditions
+  if (
+    (input == "scissors" && computerChoice == 3) ||
+    (input == "paper" && computerChoice == 1) ||
+    (input == "stone" && computerChoice == 2) ||
+    (input == "reverse scissors" && computerChoice == 2) ||
+    (input == "reverse paper" && computerChoice == 3) ||
+    (input == "reverse stone" && computerChoice == 1)
+  ) {
+    console.log(input);
+    console.log(computerChoice);
+    console.log("lost!");
+    numOfUserLosses += 1;
+    numOfComWins += 1;
+    totalNumOfRounds += 1;
+    message += `<br>${userName}, You chose ${input}. <br><br> You lose! Bummer.<br><br> (Number of Computer wins : Number of User losses) = ${numOfComWins}:${numOfUserLosses}. <br><br> so far ${userName}, you've been lossing ${numOfUserLosses}/${totalNumOfRounds} turns. Try harder!`;
+    return message;
+  }
+};
+
+var getComputerchoice = function () {
+  // Generate a decimal from 0 to 3, inclusive of 0 and exclusive of 3
+  var randomDecimal = Math.random() * 3;
+
+  // Remove the decimal
+  var randomInteger = Math.floor(randomDecimal);
+
+  // Add 1 to get valid number from 1 to 3
+  var actualNum = randomInteger + 1;
+
+  return actualNum;
```

Comment:
might be neater if this function returned the actual choice of 'scissors'/'paper'/'stone' instead of just the number :)